examples/events/index.html

```diff
@@ -31,8 +31,8 @@
     </noscript>
 
     <script src="https://unpkg.com/dicomweb-client"></script>
-    <script src="https://unpkg.com/dicom-microscopy-viewer"></script>
-    <!-- <script src="../../build/dicom-microscopy-viewer.js"></script> -->
+    <!-- <script src="https://unpkg.com/dicom-microscopy-viewer"></script> -->
```

Comment:
I don't know if you've run into trouble with this, but @swederik mentioned that it helped to specify a version for the unpkg tags.

Reply:
(Just mentioning that as an FYI - the demo works great for me)

Reply:
Thank you @pieper for the feedback. We've got this issue as well when we're deploying the event-strategy and we fixed @latest version there.

For this sample, luckily I got no error :)